app/javascript/legacy_react/src/components/common/Modal.tsx

```diff
@@ -38,7 +38,7 @@ class Modal extends React.Component<ModalProps, {}> {
             {this.props.showCloseButton ?
               <Column colSpan={1} breakSize={'xs'}>
                 <div style={{ textAlign: 'right' }}>
-                  <DefaultCloseButton onClick={() => this.props.onClose()} />
+                  <DefaultCloseButton onClick={() => {if(this.props.onClose)this.props.onClose()}} />
```

Comment:
This should be formatted on multiple lines but I think this is generally correct.